### PR TITLE
Review/final edits before release

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This is a PR with minor changes as a result of my final review.

@fivetran-reneeli feel free to merge this into `main` if all looks good on your end. Also happy to address any comments in this PR before merging. This PR contains some of my review notes directly applied to allow for easier application to the main repo.

### CODE REVIEW

File: models/twilio.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ models:
         description: Total number of read messages for this phone number. 
       - name: total_messages
         description: Total number of messages for this phone number. 
-      - name: total_spent
+      - name: total_spend
```
Comment (fivetran-joemarkiewicz) [inline models/twilio.yml:42]: I wanted to rename these fields to be `spend` as it was more reflective of the metric.

File: models/twilio__account_overview.sql [hunk 2]
```diff
@@ -7,6 +7,12 @@ with messages as (
     from {{ ref('int_twilio__messages')}}
 ),
 
+account_history as (
+
+    select * 
+    from {{ var('account_history')}}
+),
```
Comment (fivetran-joemarkiewicz) [inline models/twilio__account_overview.sql:14]: I noticed we didn't bring in the account history staging model to the account overview model. I brought it in to add some complimentary dimensions.

File: models/twilio__account_overview.sql [hunk 3]
```diff
@@ -15,9 +21,13 @@ usage_record as (
 
 select
     messages.account_id,
+    account_history.friendly_name as account_name,
+    account_history.status as account_status,
+    account_history.type as account_type,
     messages.date_day,
     messages.date_week,
     messages.date_month,
+    messages.price_unit,
```
Comment (fivetran-joemarkiewicz) [inline models/twilio__account_overview.sql:30]: For maintenance and ease of understanding, I wanted to keep the group by fields all together.

File: models/twilio__account_overview.sql [hunk 4]
```diff
@@ -35,12 +45,14 @@ select
     {% endfor %}
     
     count(messages.message_id) as total_messages,
-    round( cast(sum(messages.price) as {{ dbt.type_numeric ()}}), 2) as total_messages_spent,
-    messages.price_unit,
-    round( cast(sum(usage_record.price) as {{ dbt.type_numeric ()}}), 2) as total_account_spent
+    round( cast(sum(messages.price) as {{ dbt.type_numeric ()}}), 2) * -1 as total_messages_spend,
```
Comment (fivetran-joemarkiewicz) [inline models/twilio__account_overview.sql:48]: It read a bit strange with the spend metric for messages being negative. To make it more easily interpreted, I wanted to inverse the totals.

File: models/twilio__message_enhanced.sql [hunk 5]
```diff
@@ -48,14 +48,14 @@ final as (
...
-        messaging_service.friendly_name,
-        messaging_service.inbound_method,
-        messaging_service.us_app_to_person_registered,
-        messaging_service.use_inbound_webhook_on_number,
-        messaging_service.use_case
+        , messaging_service.friendly_name
+        , messaging_service.inbound_method
+        , messaging_service.us_app_to_person_registered
+        , messaging_service.use_inbound_webhook_on_number
+        , messaging_service.use_case
```
Comment (fivetran-joemarkiewicz) [inline models/twilio__message_enhanced.sql:58]: If the messaging service is disabled, the query will fail due to the trailing comma. This has been fixed with the above update.

Comment (fivetran-reneeli) [reply models/twilio__message_enhanced.sql:58]: Thanks Joe for catching this, I must've not ran with the variables disabled. Just as an extension, I added an additional dbt run, with variable to false, in integration testing to both source and transforms.

Comment (fivetran-reneeli) [reply models/twilio__message_enhanced.sql:58]: Oh my bad-- just realized you added it already 👍

**Pull request merged**